framework/src/timesteppers/TimeSequenceStepperBase.C

```diff
@@ -0,0 +1,101 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#include "TimeSequenceStepperBase.h"
+#include "FEProblem.h"
+#include "Transient.h"
+
+template<>
+InputParameters validParams<TimeSequenceStepperBase>()
+{
+  InputParameters params = validParams<TimeStepper>();
+  return params;
```

Comment:
can we add number of time steps between two time knots, like a uniform refinement in space? It could be useful for some convergence study.